schema/brain/queries/writes.py

```diff
@@ -6,8 +6,12 @@
 from ..connection import rethinkdb as r
 from .decorators import wrap_connection
 from .decorators import wrap_rethink_errors
-from . import RPX, RBT, RBJ, RPC, RPP
-from .reads import plugin_exists, get_plugin_by_name_controller, get_ports_by_ip_controller
+from . import RPX, RBT, RBJ, RPC, RPP, RBO
+from .reads import plugin_exists, get_plugin_by_name_controller, get_ports_by_ip_controller, get_job_by_id
+
+VALID_STATES = frozenset([
```

Comment:
see jobs.py and test_job_states.py for brain implementation of job state transitions.